src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Program.cs

```diff
@@ -34,7 +35,7 @@ public static void Main(string[] args)
 
         public static async Task MainAsync(string[] args)
         {
-            var logLevel = LogLevel.Information;
+            var trace = Trace.Messages;
```

Comment:
Why the switch from `Trace` to `LogLevel`?

Reply:
We don't want the client to own the logic of Trace -> LogLevel conversion.